hudi-client/hudi-client-common/src/test/java/org/apache/hudi/io/storage/TestHoodieHFileReaderWriter.java

```diff
@@ -198,10 +200,10 @@ public void testWriteReadHFileWithMetaFields(boolean populateMetaFields, boolean
     }
   }
 
+  @Disabled("Disable the test with evolved schema for HFile since it's not supported")
+  @ParameterizedTest
   @Override
-  @Test
-  public void testWriteReadWithEvolvedSchema() throws Exception {
-    // Disable the test with evolved schema for HFile since it's not supported
+  public void testWriteReadWithEvolvedSchema(String evolvedSchemaPath) throws Exception {
```

Comment:
should we just remove it for now? there's already a tracking jira.

Reply:
it's disabled with a message. should be a good reminder there.